tests/app/Infrastructure/Controller/PostOpenWalletControllerTest.php

```diff
@@ -39,7 +39,7 @@ public function ifUserIdNotFoundThrowsError()
     /**
      * @test
      */
-    public function ifBadUserIdThrowsBadRequest()
+    public function ifNotUserIdThrowsBadRequest()
```

Comment:
You have extracted the responsability of the the custom request to another file.
Do you think these test are in this class `tests/app/Infrastructure/Controller/PostOpenWalletControllerTest.php`??